GOVERNANCE.md

```diff
@@ -2,94 +2,171 @@
+- **Being a positive member of our community** - Go above and beyond our Code of Conduct, and commit to healthy communication in pull requests, issue discussions, Discord conversations, and interactions outside of our community (ex: no Twitter bullies allowed :)
+
+Each level unlocks new privileges and responsibilities on Discord and GitHub. Below is a summary of each contributor level:
+
+### Level 1 - Contributor
 
-**Anything that supports the Astro community is a contribution to the project.** This includes but is not limited to:
+Have you done something (big or small) to contribute to the health, success, or growth of Astro? Congratulations, you're officially recognized as a contributor to the project!
 
-- Submitting (and Merging) a Pull Request
-- Filing a Bug Report or Feature Request
-- Updating Documentation
-- Answering questions about Astro on GitHub or Discord
+#### Examples of recognized contributions
+
+- **GitHub:** Submitting a merged pull request
+- **GitHub:** Filing a detailed bug report or RFC
+- **GitHub:** Updating documentation!
+- Helping people on GitHub, Discord, etc.
 - Answering questions on Stack Overflow, Twitter, etc.
-- Blogging, Podcasting, or Livestreaming about Astro
+- Blogging, Vlogging, Podcasting, and Livestreaming about Astro
+- This list is incomplete! Similar contributions are also recognized.
 
-## Membership Levels, Roles and Responsibilities
+#### Privileges
 
-A list of all active members is available on our project README.
+- New role on [Discord](https://astro.build/chat): `@contributor`
+- New name color on Discord: **light blue**.
+- Access to exclusive Astro emotes on Discord.
+- Invitations to contributor-only events, sticker drops, and the occasional swag drop.
 
-### Contributor L1
+#### Responsibilities
 
-Have you done something to contribute to the health, success, or growth of Astro? Congratulations, you're officially a contributor!
+This role does not require any extra responsibilities or time commitment. We hope you stick around and keep participating!
 
-**Benefits:**
+If you're interested in reaching the next level and becoming a Maintainer, you can begin to explore some of those responsibilities in the next section.
 
-- Contributor status on the [Astro Discord server](https://astro.build/chat)
-- Ability to [vote](GOVERNANCE.md#voting) on some project decisions
+#### Nomination Process
 
-**Nomination:**
+_Note: This process is still in progress, and the Discord bot that will power it is not yet built. For now, manually nominate/self-nominate by posting in Discord._
 
-- Self-nominate by running `!contribute` in our Discord and briefly describe your qualifying contribution (link preferred).
+- Self-nominate by running `!contribute` in any Discord channel and briefly describe your qualifying contribution (link recommended).
```

Comment:
I know it says “any Discord channel” but is there a recommended place to do it? I don’t want to disrupt other convos going on.

”Hey, Fred, did you see this cool—” ”`!contribute`!!” ”… anyways did you see …”